chore: archive repository (#109)
traefik/traefik-hub-helm-chart: README.md

<a name="readme-top"></a>

> > Since the end of May 2024, Traefik Hub and Traefik Proxy are installed with the same Traefik Helm Chart.

Reviewer: ```suggestion
> Since the end of May 2024, Traefik Hub v2 has been deprecated and both Traefik Hub v3 and Traefik Proxy are installed with the same Traefik Helm Chart.
```

Author: I'm not sure it should sentenced that way.

> > Since the end of May 2024, Traefik Hub and Traefik Proxy are installed with the same Traefik Helm Chart.
> > This Chart is in [traefik/traefik-helm-chart](https://github.com/traefik/traefik-helm-chart)

Reviewer 2: Simplified it + continuous past perfect tense adjustment:

```suggestion
> Since the end of May 2024, Traefik Hub and Traefik Proxy have been using the same [Traefik Helm Chart](https://github.com/traefik/traefik-helm-chart) for installation.
```
